hikaricp/README.md

```diff
@@ -0,0 +1,61 @@
+# Agent Check: hikaricp
+
+## Overview
+[Hikaricp](https://github.com/brettwooldridge/HikariCP/) is a is a very lightweight and lightning-fast JDBC connection pooling framework.
+This check monitors hikaricp through the Datadog Agent.
+
+## Setup
+
+### Installation
+
+To install the hikaricp check on your host:
+
+
+1. Install the [developer toolkit]
+(https://docs.datadoghq.com/developers/integrations/new_check_howto/#developer-toolkit)
+ on any machine.
+
+2. Run `ddev release build hikaricp` to build the package.
+
+3. [Download the Datadog Agent][2].
+
+4. Upload the build artifact to any host with an Agent and
+ run `datadog-agent integration install -w
+ path/to/hikaricp/dist/<ARTIFACT_NAME>.whl`.
+
+### Configuration
+
+1. Edit the `hikaricp/conf.yaml` file, in the `conf.d/` folder at the root of your Agent's configuration directory to start collecting your hikaricp performance data. See the [sample hikaricp/conf.yaml][4] for all available configuration options.
+
+### Validation
+
+<Steps to validate integration is functioning as expected>
+
+## Data Collected
+
+### Metrics
+
+hikaricp does not include any metrics.
+
+### Service Checks
+
+hikaricp does not include any service checks.
```

Comment:
Same as the metrics section -- it looks like there is a service check associated with this integration

Reply:
Fix, thank you